server/pom.xml

```diff
@@ -393,10 +393,6 @@
                     </container>
                     <extraDirectories>
                         <paths>
-                            <path>
```

Comment:
Need to preserve the empty folder for jdbc.

Reply:
OK

Reply:
<img width="391" alt="image" src="https://github.com/apache/incubator-seata/assets/24457941/761de91d-84c4-4f5e-9cba-d39728b63cfc">
done.